webapp/src/components/team_sidebar/team_sidebar.test.tsx

```diff
@@ -32,18 +35,43 @@ const mockTheme = {
     codeTheme: 'solarized-dark',
 };
 
-jest.mock('../sidebar_buttons', () => ({
-    __esModule: true,
-    default: () => <div data-testid='mocked-sidebar-buttons'/>,
-}));
+const mockSlice = createSlice({
+    name: 'mock-reducer',
+    initialState: {
+        'plugins-com.github.manland.mattermost-plugin-gitlab': {
+            connected: true,
+            username: '',
+            clientId: '',
+            lhsData: {
+                reviews: [],
+                yourAssignedPrs: [],
+                yourAssignedIssues: [],
+                todos: [],
+            },
+            gitlabURL: '',
+            organization: '',
+            rhsPluginAction: () => true,
```

Comment:
Not sure why `clientId` is here exposed to the client. That's a bit concerning 😅  At least there's not a `clientSecret` which would be detrimental

If we are setting `connected: true`, then I think we should also set `username` and `gitlabURL`, since those will always be present when `connected` is `true`. No worries if it doesn't change the snapshot I suppose

Reply:
@mickmister , I have set values to the `gitlabURL` and `username` and updated the snapshots.

Kindly take a look at it.